library/alloc/src/fmt.rs

```diff
@@ -83,7 +83,7 @@
 //! # Formatting Parameters
 //!
 //! Each argument being formatted can be transformed by a number of formatting
-//! parameters (corresponding to `format_spec` in the syntax above). These
+//! parameters (corresponding to `format_spec` in the syntax below). These
```

Comment:
Is there a way to link to an anchor on the same page? If so, how about changing this to just "syntax" and linking it?

Reply:
Thanks for suggestion, replaced with a link to the syntax anchor.